barcode/tests/__init__.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from barcode import *
+from barcode_utils import generate_barcode_from_badge_num, get_badge_num_from_barcode
+
+
+@pytest.fixture
+def cfg(monkeypatch):
+    config['secret'] = dict()
+    config['secret']['barcode_key'] = "ABCDEF1234"
+    config['secret']['barcode_salt'] = 42
+    config['secret']['barcode_event_id'] = 0xFF
+
+
+def test_cfg_fixtures(cfg):
```

Comment:
Testing for valid config options is a good idea, but it won't really catch bad config options unless you explicitly run the unit tests after you do the configuration. Since the unit tests are really just run before merges, and the config options are usually edited after we merge the code, these kinds of assertions should probably be in the main plugin.